airbyte-lib/examples/run_spacex.py

```diff
@@ -11,16 +11,20 @@
 # In separate terminal:
 #   poetry run python examples/run_spacex.py
 
-source = ab.get_connector("source-spacex-api", config={"id": "605b4b6aaa5433645e37d03f"})
-cache = ab.get_in_memory_cache()
+source = ab.get_connector(
+    "source-spacex-api",
+    config={"id": "605b4b6aaa5433645e37d03f"},
+    install_if_missing=True,
+)
+cache = ab.new_local_cache(source_catalog=source.configured_catalog)
 
 source.check()
 
 source.set_streams(["launches", "rockets", "capsules"])
 
 result = source.read_all(cache)
 
-print(islice(source.read_stream("capsules"), 10))
+print(islice(source.get_stream_records("capsules"), 10))
 
 for name, records in result.cache.streams.items():
```

Comment:
It seems like those are outdated - `result` doesn't have a direct `cache` member anymore

Reply:
I didn't retest these `examples/run_*.py` scripts - I only made small updates to match naming.

@flash1293 - Should we update these so they still work as expected? Are they helpful?

Reply:
Done. Added back "cache" property for ReadResult (previously SyncResult).